src/SecureSign.Core/Signers/AuthenticodeSigner.cs

```diff
@@ -61,7 +61,14 @@ public async Task<byte[]> SignAsync(byte[] input, X509Certificate2 cert, string
 
 				if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
 				{
-					return await SignUsingSignToolAsync(inputFile, certFile, password, description, url);
+                    if (fileExtention.Contains("ps"))
```

Comment:
You probably want to check for PowerShell extensions specifically (`.ps1` etc) - Searching just for `ps` anywhere in the extension might have false positives?

Reply:
Actually this was on purpose as the Set-Authenticode command will work for everything sent it's way from the Authenticode Endpoint it is already in, by keeping it as ps you open up ps1, psm1, psd1 etc.. and anything else like exe, dlls, cabs etc would not fail either.. not that it would hit anyway.